config/locales/en.yml

```
@@ -36,7 +36,10 @@ en:
       email_messages: "E-mail Message Settings"
       email_messages_enabled: "Enable e-mail notifications"
       email_messages_submit: "Update"
+      collections: "Groups"
   works:
+    collection: "Group"
+    collection_tags: "Group Tags"
```

Are these really collection tags still?  If not then the key should change to group
```suggestion
    group: "Group"
    group_tags: "Group Tags"
```